examples/tv-casting-app/linux/main.cpp

```diff
@@ -114,6 +115,30 @@ HelpOptions helpOptions("tv-casting-app", "Usage: tv-casting-app [options]", "1.
 
 OptionSet * allOptions[] = { &cmdLineOptions, &helpOptions, nullptr };
 
+static void MyBoundDeviceChangedHandler(const EmberBindingTableEntry & binding, chip::DeviceProxy * peer_device, void * context)
```

Comment:
`My` prefix is a bit vague. Handlers are often named `On...`, so this could be `OnBoundDeviceChanged`

Reply:
fixed name